homeassistant/components/plugwise/sensor.py

```diff
@@ -353,8 +353,8 @@ def __init__(
             self._entity_name = f"Smile {self._entity_name}"
 
     @callback
-    def _async_process_data(self) -> None:
-        """Update the entity."""
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
         if not (data := self.coordinator.data.devices.get(self._dev_id)):
             LOGGER.error("Received no data for device %s", self._entity_name)
             self.async_write_ha_state()
```

Comment:
There's another occurrence below on line 364.
```suggestion
            super()._handle_coordinator_update()
```